doc/releasenotes/14_0_0_summary.md

```diff
@@ -88,6 +90,42 @@ API endpoint `/throttler/throttle-app` now accepts a `ratio` query argument, a f
 - `1` means "always throttle"
 - any numbr in between is allowd. For example, `0.3` means "throttle in 0.3 probability", ie on a per request and based on a dice roll, there's a `30%` change a request is denied. Overall we can expect about `30%` of requests to be denied. Example: `/throttler/throttle-app?app=vreplication&ratio=0.25`
 
+See new SQL syntax for controlling/viewing throttling, down below.
+
+### New Syntax
+
+#### Control and view Online DDL throttling
+
+We introduce the following syntax, to:
+
+- Start/stop throttling for all Online DDL migrations, in general
+- Start/stop throttling for a particular Online DDL migration
+- View throttler state
+
+
+```sql
+ALTER VITESS_MIGRATION '<uuid>' THROTTLE [EXPIRE '<duration>'] [RATIO <ratio>];
+ALTER VITESS_MIGRATION THROTTLE ALL [EXPIRE '<duration>'] [RATIO <ratio>];
+ALTER VITESS_MIGRATION '<uuid>' UNTHROTTLE;
+ALTER VITESS_MIGRATION UNTHROTTLE ALL;
+SHOW VITESS_THROTTLED_APPS;
+```
+
+default `duration` is `24h`
```

Comment:
Any reason for this default?  It seems less foot-gun-prone if we make this very long (~infinite?) by default, and then if you really want it to expire, you specify something here.

Reply:
I feel like timeouts are a good idea in general. I have no strong opinions on the particular default value. I could make it 10 years.

Reply:
expire's default now set as 100 years, which we consider as "infinite".